src/components/ChatGhostUI/ContentLoadingOneLine.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {Rect, Circle} from 'react-native-svg';
+import ContentLoadingWrapper from './ContentLoadingWrapper';
+
+const ContentLoadingOneLine = () => (
```

Comment:
**suggestion:** `ReportActionsSkeletonView`

**suggestion:** Don't create 3 separate components that all do basically the same thing. Have one component and specify the number of lines as a prop.

Reply:
Done